bindings/wasm/manifold.d.ts

```diff
@@ -40,21 +48,36 @@ type Curvature = {
 };
 
 declare class Mesh {
+  constructor(options: {
+    numProp: number,
+    vertProperties: Float32Array,
+    triVerts: Uint32Array,
+    mergeFromVert?: Uint32Array,
+    mergeToVert?: Uint32Array,
+    runIndex?: Uint32Array,
+    runOriginalID?: Uint32Array,
+    faceID?: Uint32Array,
+    halfedgeTangent?: Float32Array,
+    runTransform?: Float32Array
+  });
   numProp: number;
   vertProperties: Float32Array;
   triVerts: Uint32Array;
   mergeFromVert?: Uint32Array;
   mergeToVert?: Uint32Array;
   runIndex?: Uint32Array;
-  originalID?: Uint32Array;
+  runOriginalID?: Uint32Array;
   faceID?: Uint32Array;
   halfedgeTangent?: Float32Array;
+  runTransform?: Float32Array;
   get numTri(): number;
   get numVert(): number;
-  verts(tri: number): Uint32Array<3>;
-  position(vert: number): Float32Array<3>;
+  get numMatrix(): number;
+  verts(tri: number): SealedUint32Array<3>;
+  position(vert: number): SealedFloat32Array<3>;
   extras(vert: number): Float32Array;
-  tangent(halfedge: number): Float32Array<4>;
+  tangent(halfedge: number): SealedFloat32Array<4>;
+  matrix(index: number): SealedFloat32Array<12>;
```

Comment:
Looks like this and `numMatrix` above need their names updated. Nit: I'd call the input `run` instead of `index`.

Reply:
Did you miss this comment by chance?

Reply:
I did, my bad. It should be fixed now